go.mod

```diff
@@ -27,25 +27,27 @@ require (
 	github.com/pkg/xattr v0.4.7
 	github.com/prometheus/client_golang v1.12.2
 	github.com/stretchr/testify v1.8.1
-	golang.org/x/crypto v0.0.0-20220315160706-3147a52a75dd
+	golang.org/x/crypto v0.1.0
 	golang.org/x/net v0.1.0
 	golang.org/x/sys v0.1.0
 	google.golang.org/grpc v1.49.0
 	google.golang.org/protobuf v1.28.0
-	k8s.io/api v0.25.0
-	k8s.io/apimachinery v0.25.0
+	k8s.io/api v0.25.3
+	k8s.io/apimachinery v0.25.3
 	k8s.io/client-go v12.0.0+incompatible
-	k8s.io/cloud-provider v0.25.0
+	k8s.io/cloud-provider v0.25.3
 	k8s.io/klog/v2 v2.80.1
 	//
 	// when updating k8s.io/kubernetes, make sure to update the replace section too
 	//
-	k8s.io/kubernetes v1.25.0
-	k8s.io/mount-utils v0.25.0
+	k8s.io/kubernetes v1.25.3
+	k8s.io/mount-utils v0.25.3
 	k8s.io/utils v0.0.0-20220728103510-ee6ede2d64ed
 	sigs.k8s.io/controller-runtime v0.11.0-beta.0.0.20211208212546-f236f0345ad2
 )
 
+require k8s.io/pod-security-admission v0.0.0
```

Comment:
actually, I do not really like it that this is on it's own line. It should be part of the block above.

Reply:
addressed this in https://github.com/ceph/ceph-csi/pull/3485#pullrequestreview-1163027598, lets rebase this once that is merged.

Reply:
aborted all CI jobs, no need to test this PR atm